project/src/components/app/app.jsx

```diff
@@ -8,16 +8,27 @@ import FavoritesPage from '../favorites-page/favorites-page.jsx';
 import RoomPage from '../room-page/room-page.jsx';
 import HotFoundPage from '../not-found-page/not-found-page.jsx';
 import offerType from '../offers-prop/offers-prop.js';
-
+import cityType from '../city-prop/city-prop.js';
 function App(props) {
-  const { rentalOfferCout, offers } = props;
+  const { rentalOfferCout, offers, defaultCity } = props;
+  const [selectedOffer, setSelectedOffer] = useState({});
```

Comment:
```suggestion
  const [selectedOffer, setSelectedOffer] = useState(null);
```
Пустые объекты обычно не ставят

Отдавай предпочтение `null` в таких ситуациях